src/Simulation/EntryPointDriver/Azure.cs

```diff
@@ -2,33 +2,32 @@
 // Licensed under the MIT License.
 
 using System;
-using System.CommandLine.Parsing;
 using System.Linq;
 using System.Threading.Tasks;
 using Microsoft.Azure.Quantum;
 using Microsoft.Azure.Quantum.Exceptions;
 using Microsoft.Quantum.Runtime;
 using Microsoft.Quantum.Simulation.Common.Exceptions;
+using Microsoft.Quantum.Simulation.Core;
 using static Microsoft.Quantum.EntryPointDriver.Driver;
 
 namespace Microsoft.Quantum.EntryPointDriver
 {
     /// <summary>
     /// Provides entry point submission to Azure Quantum.
     /// </summary>
-    internal static class Azure
+    public static class Azure
```

Comment:
I saw the conversation below mentioning that System.CommandLine needs the class and all properties to be public. Could you elaborate why that is? I.e. what changed to enable multiple entry points that requires AzureSettings to be public when it didn't need to be before?

Reply:
I think this is getting confused with the AzureSettings class. That class has to have public members for the System.CommandLine to properly populate its members, and it had public members before, so no change there. What has changed is that the settings class, as well as this Azure class have had to become public so they could be directly referenced by the generated entry points. This was not needed before when the Driver class was generic, but removing the type parameter from the Driver class made it possible to have multiple entry points.

Reply:
Got it. I was referring to the AzureSettings property in the class, but that answers my question.